styles/pages/index.less

```diff
@@ -135,7 +134,6 @@ section.faq {
   .img {
     display: none;
   }
-
```

Comment:
Is this just removing a newline? In my expeirence style adjustments usually go in their own commit / PR.